backend/main/models/community.go

```diff
@@ -156,8 +157,8 @@ func GetCommunitiesForHomePage(db *s.Database, start, count int) ([]*Community,
     	WHERE status = 'published' AND end_time < (NOW() AT TIME ZONE 'UTC')
     	GROUP BY community_id
     	HAVING COUNT(*) >= 2
-  	)
-	OR id = 1
+  	))
+	OR is_featured = 'true'
```

Comment:
@dbslone For this query, did we only want to return featured communities, OR keep the existing logic as well, OR, only return communities that match the existing logic AND are featured.

Reply:
This is good I can update the Flow community to have the featured flag so it appears